downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md

```diff
@@ -6,29 +6,39 @@ title: Swan Lake Preview 1 Release note
 This release is the first preview version of Ballerina Swan Lake. This release includes a new set of Language features and significant improvements to the compiler, runtime, standard libraries, and developer tooling.
  
 # Highlights
-- Highlight I
-- Highlight II
+
+- New recursive descent parser
+- Type system enhancements
+- Improved language integration support
+- Improvements on Standard Library modules 
+- Improved mocking support in testing
 
 # What's new in Ballerina Swan Lake - Preview 1
+
 ## Language
+
 The Language implementation is based on [Ballerina Language Specifications Draft 2020-06-18](https://ballerina.io/spec/lang/draft/v2020-06-18/). This Specification introduces a set of new features and improvements in the following main areas.   
  
-- Type system enhancements (e.g. New Error and Table Design) 
-- Improved Immutability support 
-- [New Transaction support] (https://github.com/ballerina-platform/ballerina-spec/blob/master/lang/proposals/transaction/transaction.md) 
-- Improved Query support 
+- Type system enhancements (e.g., new error and table design) 
```

Comment:
I have updated this section release note.

Reply:
Updated those.